core/block_validator.go

```diff
@@ -141,7 +141,8 @@ func (v *BlockValidator) ValidateState(block *types.Block, statedb *state.StateD
 				return err
 			}
 			statedb.Finalise(v.config.IsEIP158(header.Number))
-			statedb.AccountsIntermediateRoot()
+			//state verification pipeline - accounts root are not calculated here
+			statedb.PopulateSnapAccountAndStorage()
```

Comment:
Please measure how long it will takes, if it takes too long, we may make it parallel

Reply:
sure